src/library_wasmfs.js

```diff
@@ -85,6 +86,37 @@ var WasmfsLibrary = {
       // For file preloading, cwd should be '/' to begin with.
       return '/';
     }
+  },
+  emscripten_wasmfs_write_buffer : function(index, buffer, length, offset) {
```

Comment:
This name suggests it is a public API, which I think it isn't. As an internal API, please prefix it with `_`.

Second, "write buffer" is a little unclear to me. Why mention "buffer" here? I'd expect something else in the name actually, a mention that it is writing to a JS memory file in particular.

```suggestion
  emscripten_wasmfs_write_buffer: function(index, buffer, length, offset) {
```

Reply:
Done.